pkg/server/event_handler.go

```diff
@@ -353,31 +353,31 @@ func parseGitHubSource(ctx context.Context, data []byte, objAttrs map[string]str
 	// Set github-commit.
 	c, found := pm.Metadata[MetadataKeyGitHubCommit]
 	if !found {
-		logger.Infof("github-commit not found")
+		logger.Infof("MetadataKeyGitHubCommit not found")
```

Comment:
Looks like you have a misunderstanding about my previous comment. I mean using the constant here https://github.com/abcxyz/pmap/blob/d001c0d517e11b16284ab36b5fc2d0d0fca0ac81/pkg/server/event_handler.go#L78 . Not using `MetadataKeyGitHubCommit` as a raw string.

Reply:
something like 
```
logger.Infof("%s not found", MetadataKeyGitHubCommit)
```

Reply:
Updated.